src/csv_parser.c

```diff
@@ -0,0 +1,309 @@
+	* of time stamp 1 (csv[u_ts[0]][0]) */
+	printf("\nSensor readings for timestamp1 {%f} stored in sr[1][]"
+			"array in row 1 (ranging from 0 to time stamp2" 
+			" counter sr_c[1]) is\n",csv[u_ts[0]][0]);
+	for(int i=0;i<sr_c[1];i++) {
+		printf("%f \n",sr[1][i]);
+	}
+	/* Print row 2 of sr[2][] array to display sensor readings 
+	* of time stamp 2 (csv[u_ts[1]][0]) */
+	printf("\nSensor readings for timestamp2 {%f} stored in sr[2][]"
+		  "array in row 2 (ranging from 0 to time stamp3"
+		  " counter sr_c[2]) is\n", csv[u_ts[1]][0]);
+	for(int i=0;i<sr_c[2];i++) {
+		printf("%f \n",sr[2][i]);
+	}
+	/* Print row 3 of sr[3][] array to display sensor readings 
+	* of time stamp 3 (csv[u_ts[2]][0]) */
+	printf("\nSensor readings for timestamp3 {%f} stored in sr[3][]"
+		    "array in row 3 (ranging from 0 to time stamp4" 
+		    " counter sr_c[3]) is\n", csv[u_ts[2]][0]);
+	for(int i=0;i<sr_c[3];i++) {
+		printf("%f \n",sr[3][i]);
+	}
+		/* Print row 4 of sr[4][] array to display sensor readings 
+	* of time stamp 4 (csv[u_ts[3]][0]) */
+	printf("\nSensor readings for timestamp4 {%f} stored in sr[4][]"
+		    "array in row 4 (ranging from 0 to time stamp5" 
+		    " counter sr_c[4]) is\n", csv[u_ts[3]][0]);
+	for(int i=0;i<sr_c[4];i++) {
+		printf("%f \n",sr[4][i]);
+	}
+    /* Print row 5 of sr[5][] array to display sensor readings 
+	* of time stamp 5 (csv[u_ts[4]][0]) */
+	printf("\nSensor readings for timestamp5 {%f} stored in sr[5][]"
+				"array in row 5 (ranging from 0 to time stamp6" 
+				" counter sr_c[5]) is\n", csv[u_ts[4]][0] );
+	for(int i=0;i<sr_c[5];i++) {
+		printf("%f \n",sr[5][i]);
+	}
+	  /* Print row 6 of sr[6][] array to display sensor readings 
+	* of time stamp 6 (csv[u_ts[5]][0]) */
+	printf("\nSensor readings for timestamp6 {%f} stored in sr[6][]" 
+			"array in row 6 (ranging from 0 to time stamp7" 
+			" counter sr_c[6]) is\n", csv[u_ts[5]][0]);
+	for(int i=0;i<sr_c[6];i++) {
+		printf("%f \n",sr[6][i]);
+	}
+	  /* Print row 7 of sr[7][] array to display sensor readings 
+	* of time stamp 7 (csv[u_ts[6]][0]) */
+	printf("\nSensor readings for timestamp7 {%f} stored in sr[7][]"
+			"array in row 7 (ranging from 0 to time stamp8" 
+			" counter sr_c[7]) is\n", csv[u_ts[6]][0]);
+	for(int i=0;i<sr_c[7];i++) {
+		printf("%f \n",sr[7][i]);
+	}
+	/* Print row 8 of sr[8][] array to display sensor readings 
+	* of time stamp 8 (csv[u_ts[7]][0]) */
+	printf("\nSensor readings for timestamp8 {%f} stored in sr[8][]"
+			"array in row 8 (ranging from 0 to time stamp9" 
+			" counter sr_c[8]) is\n", csv[u_ts[7]][0]);
+	for(int i=0;i<sr_c[8];i++) {
+		printf("%f \n",sr[8][i]);
+	}
+	/* Print row 9 of sr[9][] array to display sensor readings 
+	* of time stamp 9 (csv[u_ts[8]][0]) */
+	printf("\nSensor readings for timestamp9 {%f} stored in sr[9][]"
+			"array in row 9 (ranging from 0 to time stamp10"
+			" counter sr_c[9]) is\n", csv[u_ts[8]][0]);
+	for(int i=0;i<sr_c[9];i++) {
+		printf("%f \n",sr[9][i]);
+	}	
```

Comment:
Same comment as above, this is a lot of repeat code that doesn't really belong in a main function. It should be its own function.

Reply:
This was few print statements to convey the function results, and this will not be included in the main modules. Removed the print lines in the latest dev branch pull request.